src/rust/dbv2/README.md

```diff
@@ -0,0 +1,28 @@
+# Database
+
+This repository provides migrations to create the Econia event database, and Rust types to work with the event database.
+
+The aggregator furthermore adds tables to this database under the `aggregator` schema. Those are disucssed in the `src/rust/aggregator` crate.
+
+# Contributing
+
+When creating a new migration, before committing, please make sure to use [sql-formatter](https://github.com/sql-formatter-org/sql-formatter) with the provided configuration.
+
+Also make sure to add the corresponding documentation to `models.md`.
```

Comment:
I left some comments on the `models.md` file, and toward the end of review realized we might want to get right of that file (see comments there)

Essentially it is cleaner if we just point developers to the source or DocGen files, rather than creating duplicate work there to maintain

Hence if we take it out, you miight want to update the description here with a simple reference to the corresponding Rust source

Reply:
So the `models.md` file has been taken out, but there is still a reference to it here.

How about taking out the reference, here, to `models.md`?